packages/build-utils/src/index.mjs

```diff
@@ -0,0 +1 @@
+export { default as viteConfig } from './vite.config.mjs';
```

Comment:
the use of mjs threw me off here, since after chatting with you, you clarified that this is an outlier and most packages can stick with .ts and .js files and extensionless imports as terra-ui has done for some time.

Let's have a small README in this package's root folder to call this out and have an intro blurb on the intended use of the package.

Reply:
Done in https://github.com/DataBiosphere/terra-ui/pull/4116/commits/a95822f25ce85cb2733af8e53b84647a6a7b1cff.